.golangci.yaml

```diff
@@ -1,7 +1,7 @@
 linters-settings:
   gomoddirectives:
     replace-allow-list:
-      - github.com/trento-project/contracts
+      - github.com/trento-project/contracts/go
```

Comment:
I think we can remove the `linters-settings` block as we don't have any replace anymore

Reply:
I think @arbulu89 is right ._.

Reply:
done!